health/middlewares.js

```diff
@@ -0,0 +1,44 @@
+const nHealth = require('n-health');
+
+const { HEALTCHECK_DUMMY_USER_UUID } = require('config');
+
+const panicGuide = 'Contact the FT Accounts team #ft-accounts-team.';
+const businessImpact = 'Users will be unable to use the Syndication apps';
+
+const middlewares = [
```

Comment:
I think middlewares is a bit of misleading term here. I think you should mentions the following as `Services` and related endpoints. [See also here](https://github.com/Financial-Times/next-profile/blob/b62423a62d612c1b49c2ef27d330ce79ff71906d/server/config/health-checks/membership.js#L136).

Reply:
Changed